kitsune/products/jinja2/products/blocks/cta_block.html

```diff
@@ -0,0 +1,12 @@
+{% if value.type=='Community' %}
```

Comment:
Is this check needed since we don't have another type at the moment?

Reply:
Dropped